docs/4.4/api-reference.md

```diff
@@ -11,57 +11,328 @@ Teleport is currently working on documenting our API.
 
         We are currently working on this project. If you have an API suggestion, [please complete our survey](https://docs.google.com/forms/d/1HPQu5Asg3lR0cu5crnLDhlvovGpFVIIbDMRvqclPhQg/edit).
 
+# gRPC APIs
```

Comment:
I don't think we need to tell people it's a GRPC API. If they are using the Teleport Client they might be using HTTP under the hood anyway.